### DISCUSSION TITLE
Livewire random loose focus while Typing within an Input field

### DISCUSSION THREAD

**Discussion by HannesEURESA** [livewire/livewire#8701] [7. Bugs (archived)]
### Livewire version

3.5.4

### Laravel version

10.44.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create Textarea or Input
2. The Textarea is built with wire:model.live
3. Writing within the Textarea/Input Field causes the Problem that the cursor Jumps out of the Textarea Field.

We are creating Form Fields which are built with wire:model.live. In Some cases, not everytime, if we are typing in the Form Fields, the Cursor jumps randomly out of the Form Field for no reasons and looses the Focus. There are no Validations to trigger or to cause Jumping problems. We can´t find a possible Solution or workarround to keep the cursor inside the field.

Here are 2 Examples of Fields which causes Problems
1)`<div> <textarea class="placeholder-zinc-200 text-zinc-800 py-3 px-4 block w-full rounded-md border-zinc-300 px-3 shadow-sm focus:border-transparent focus:outline-none focus:ring focus:ring-vt-blue-500 focus:ring-opacity-75" autocomplete="off" wire:model.live="notice"></textarea></div><div class="@if ($char <= 100) text-green-500 @else text-red-500 @endif mt-1"> {{ $char }} Chars</div><div>@if ($error) <div class="mt-1 text-xs text-red-500">{{ $error }}</div>@endif</div>`

2)`<div><input type="text" wire:model.live="code" placeholder="AAAA" id="code" class="w-full rounded-md border border-zinc-300 px-4 py-1.5 uppercase text-zinc-800 placeholder-zinc-300 shadow-sm placeholder:text-sm focus:border-transparent focus:outline-none focus:ring focus:ring-blue-500 focus:ring-opacity-75" /></div><div>@if ($error)<div class="mt-1 text-xs text-red-500">{{ $error }}</div>@endif</div>`

If we are using wire.model.blur it seems to fix the Problem, but in some cases we need the .live state to trigger Input or Text Paragraphs. But then the cursor randomly jumps out of the Fields. And not everytime, it happens 2-3 times of 100 users. We get nearly daily responses of this behaviour. 

I hope someone can help us or can help to fix this with an workaround.

### Repository or Wirebox URL

https://github.com/HannesEURESA/livewireBug/tree/main/livewire_bug

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by larryx64** [comment]
Were you able to solve this? 

I've been scratching my head for quite a while and I've found [multiple](https://github.com/livewire/livewire/discussions/4790) ~issues~ [discussions](https://github.com/alpinejs/alpine/discussions/2826) on this [topic](https://github.com/alpinejs/alpine/issues/3097), but no one seems to have a proper solution...

I'm not even using `$id('my-element')` anywhere in my project. As soon as I have an input with an id, it either looses focus on every component refresh, or I get this error when submitting my form: `Uncaught Snapshot missing on Livewire component with id: VJw8Lud41LX8g7LQp9YZ`

Not sure what to do from here...

**Comment by lrljoe** [comment]
I'd strongly discourage using 'live" on a textarea, use a denounce instead.
Otherwise you are going to end up with issues.

Live is going to submit every time it changes (i.e. on key up), your server side update is going to fire, and the page will re-render.

Debounce it by 500ms, and it will behave.

Or don't, and when your site has 5,000 users typing, you'll end up with 100,000 requests per second, and will need to scale your solution out massively, needlessly.
